Makefile

```diff
@@ -0,0 +1,79 @@
+.DEFAULT_GOAL := help
+
+SHELL := /bin/bash
+
+GO111MODULE := on
+
+GOPKG += github.com/veraison/evcli/cmd/psa
+
+MOCKGEN := $(shell go env GOPATH)/bin/mockgen
+INTERFACES := common/iveraisonclient.go
+MOCKPKG := mocks
+
+GOLINT ?= golangci-lint
+
+ifeq ($(MAKECMDGOALS),lint)
+GOLINT_ARGS ?= run --timeout=3m
+else
+  ifeq ($(MAKECMDGOALS),lint-extra)
+  GOLINT_ARGS ?= run --timeout=3m --issues-exit-code=0 -E dupl -E gocritic -E gosimple -E lll -E prealloc
+  endif
+endif
+
+.PHONY: lint lint-extra
+lint lint-extra: _mocks; $(GOLINT) $(GOLINT_ARGS)
+
+ifeq ($(MAKECMDGOALS),test)
+GOTEST_ARGS ?= -v -race $(GOPKG)
+else
+  ifeq ($(MAKECMDGOALS),test-cover)
+  GOTEST_ARGS ?= -short -cover $(GOPKG)
+  endif
+endif
+
+COVER_THRESHOLD := $(shell grep '^name: cover' .github/workflows/ci-go-cover.yml | cut -c13-)
+
+define MOCK_template
+mock_$(1): $(1)
+	$$(MOCKGEN) -source=$$< -destination=cmd/mocks/$$$$(basename $$@) -package=$$(MOCKPKG)
+endef
+
+$(foreach m,$(INTERFACES),$(eval $(call MOCK_template,$(m))))
+MOCK_FILES := $(foreach m,$(INTERFACES),$(join mock_,$(m)))
+CLEANFILES := $(MOCK_FILES)
+
+_mocks: $(MOCK_FILES)
+.PHONY: _mocks
+
+.PHONY: test test-cover
+test test-cover: _mocks; go test $(GOTEST_ARGS)
+
+realtest: _mocks; go test $(GOTEST_ARGS)
```

Comment:
Is realtest needed ? I see no difference between realtest and test ?

it is also not present in the available targets?